.github/workflows/documentation.yml

```
@@ -0,0 +1,81 @@
+        with:
+          ruby-version: 3.2
+          bundler-cache: true
+          
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+          
+      - name: Setup Pages
+        id: pages
+        uses: actions/configure-pages@v3
```

actionlint (1.7.7)

the runner of "actions/configure-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)